linux_os/guide/system/permissions/partitions/mount_option_boot_nosuid/rule.yml

```diff
@@ -22,6 +22,8 @@ identifiers:
     cce@rhel7: CCE-82138-9
     cce@rhel8: CCE-81033-3
     cce@rhel9: CCE-83877-1
+    cce@sle15: CCE-91235-2
```

Comment:
Please sort these.

`./utils/fix_rules.py -y sort_subkeys` will do it automatically.